app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -204,4 +206,26 @@ def sshkeys
       persister.auth_key_pairs.build(:name => tenant_key[:name])
     end
   end
+
+  def systemtypes
+    collector.system_pool.each do |v|
+      persister.flavors.build(
+        :type    => "::ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SystemType",
```

Comment:
Minor but you don't need the leading `::` before the `ManageIQ::...etc`

That only is used to indicate a top-level class name, e.g. if you are in a namespace like
`module ManageIQ::Providers` and there is a `class ManageIQ::Providers::Flavor`, it is ambiguous when you just say `Flavor` because ruby will look to your current namespace first and find something.  If you really need the top-level class you can use `::Flavor` and it will skip your current namespace.